include/xtensor/xtensor.hpp

```diff
@@ -518,6 +518,36 @@ namespace xt
     {
         return m_storage;
     }
+
+    /**
```

Comment:
To make this doc appear somewhere the signature needs to be included in an `rst` file, right?